omnigibson/utils/transform_utils.py

```diff
@@ -534,6 +533,18 @@ def quat2euler(quat):
     """
     return R.from_quat(quat).as_euler("xyz")
 
+def wrap_angle(theta, lower=-np.pi):
+    """"
+    Converts an angle to the range [lower, lower + 2*pi).
+    Defaults to wrapping to [-pi, pi).
+
+    Args:
+        theta (float): angle in radians
+
+    Returns:
+        float: angle in radians in range [lower, lower + 2*pi)
+    """
+    return (theta - lower) % (2 * np.pi) + lower
```

Comment:
I don't love this. This kind of logic helps very little with some very bug-prone scenarios (very easy to end up with major vs minor arcs etc). The ideal scenario involves an object like scipy's rotation that supports some key operations so that you can't mess up by trying to subtract these yourself etc. - let's chat about this later. Leave this issue unresolved.

Reply:
Moved this into the planning utils as a much simpler function.